src/test/java/com/redhat/darcy/webdriver/CookieTest.java

```diff
@@ -0,0 +1,60 @@
+package com.redhat.darcy.webdriver;
+
+import static org.junit.Assert.assertNotNull;
+
+import com.redhat.darcy.web.Cookie;
+import com.redhat.darcy.web.SimpleUrlView;
+import com.redhat.darcy.web.api.Browser;
+
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Set;
+
+@RunWith(JUnit4.class)
+public class CookieTest {
+
+    @Test
+    public void shouldBeACookie() {
+        new Cookie("Fish", "cod", "", "", null, false);
+    }
+
+    @Test
+    public void shouldValidateAValidCookie() {
+        new Cookie("Fish", "cod", "", "", null, false)
+                .validate();
+    }
+
+    @Test
+    public void shouldFailValidationForACookieWithAnIllegalName() {
+        new Cookie("Fish;", "cod", "", "", null, false)
+                .validate();
+    }
+
+    @Test
+    public void shouldPrintCookieToString() {
+        System.out.println(new Cookie("Fish;", "cod", "", "", LocalDateTime.now(), false)
+                .toString());
+    }
```

Comment:
I'm not sure what the best practice is on unit testing forked code, but regardless all of the above tests are testing darcy-web, not darcy-webdriver. They should be apart of that PR, not this one. This is not testing WebDriverCookieManager, in other words. They don't use that class.